pkg/plugins/pg/README.md

```diff
@@ -1,144 +1,149 @@
+| ---------------- | ------------------------------------------------------------------------------------------------------------------------------- | -------------------- | -------------------- |
+| table            | the name of the table in Postgres that the connector should read                                                                | yes                  | n/a                  |
+| url              | formatted connection string to the database.                                                                                    | yes                  | n/a                  |
+| columns          | comma separated string list of column names that should be built in to each Record's payload.                                   | no                   | `*` (all columns)    |
+| key              | column name that records should use for their `Key` fields. defaults to the column's primary key if nothing is specified        | no                   | primary key of table |
+| snapshot         | whether or not the plugin will take a snapshot of the entire table acquiring a read level lock before starting cdc mode         | n/a                  | enabled              |
+| cdc              | enables CDC features                                                                                                            | req. for CDC mode    | off                  |
+| publication_name | name of the publication to listen for WAL events                                                                                | req. for CDC mode    | `pglogrepl`          |
+| slot_name        | name of the slot opened for replication events                                                                                  | req. for CDC mode    | `pglogrepl_demo`     |
+| replication_url  | URL for the CDC connection to use. If no replication_url is provided, then the CDC connection attempts the use the `url` value. | optional in CDC mode | n/a                  |
+
+Destination 
+===========
+The Postgres Destination takes a `record.Record` and parses it into a valid 
+SQL query. The Destination is designed to handle different payloads and keys.
+decause of this, each record is individually parsed and upserted. 
+
+## Table Name
+Every record must have a `table` property set in it's metadata, otherwise it
+will error out. However, because of this, our Destination write can support 
+multiple tables in the same connector provided the user has proper access to 
+those tables.
+
+## Keys
+Keys in the Destination are optional and must be unique if they are set.
 
-We plan to support alphanumeric and composite record keys soon.
+If a Key is included in a Payload, it will be removed.  This is because the Key 
+is also inserted into the database, so the Payload removes it. 
 
-# Roadmap 
-- [x] Change data capture handling
-- [ ] Composite key support 
-- [ ] Alphanumeric position handling 
-- [ ] JSONB support
```

Comment:
Removed because our README should no longer be a source of info for roadmap or features.